test/Riok.Mapperly.Tests/_snapshots/ObjectPropertyInitPropertyTest.RequiredPropertySourceNotFoundShouldDiagnostic.00.verified.txt

```diff
@@ -0,0 +1,28 @@
+﻿{
+  Diagnostics: [
+    {
+      Id: RMG012,
+      Title: Mapping source property not found,
+      Severity: Info,
```

Comment:
I think this should be of level error, as this can never work for required properties. Therefore a new diagnostic descriptor should be introduced (eg. `RequiredPropertyNotMapped`) with a severity of error which is reported for properties with the required flag instead of the `RMG012`.